src/System.Runtime.Extensions/src/System/Environment.WinRT.cs

```diff
@@ -0,0 +1,124 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.Foundation.Metadata;
+using Windows.Storage;
+
+namespace System
+{
+    public static partial class Environment
+    {
+        private static string GetFolderPathCore(SpecialFolder folder, SpecialFolderOption option)
+        {
+            // For testing we'll fall back if the needed APIs aren't present
+            if (ApiInformation.IsTypePresent("Windows.Storage.UserDataPaths"))
+            {
+                return GetFolderPathCoreCurrent(folder, option);
+            }
+            else
+            {
+                return GetFolderPathCoreFallBack(folder, option);
+            }
+
+        }
+
+        private static string GetFolderPathCoreCurrent(SpecialFolder folder, SpecialFolderOption option)
```

Comment:
I left it out on purpose. Didn't seem like a good thing to be trying to create folders in lowbox- worried about trying to do this and negatively impacting the system (bad ACLs, what have you). They (Windows) explicitly didn't expose the option in their new APIs.

I'll add comments.